simulator/examples/text-fonts.rs

```diff
@@ -1,3 +1,7 @@
+//! # Example: Fonts
+//!
+//! Demonstrate the available builtin fonts.
```

Comment:
If this is supposed to show all builtin fonts, we should also add the 24x32 font to the example.